### PR TITLE
stack to queue app

### PR DESCRIPTION
**Pull request by alexander-bedenko**

### CODE REVIEW

File: lab4/task1/file1.sql [hunk 1]
```diff
@@ -1 +0,0 @@
-CREATE DATABASE Something
```
Comment (pryabov) [inline lab4/task1/file1.sql]: not sure why this file deleted?

File: lab4/task1/file2.sql [hunk 2]
```diff
@@ -1,2 +0,0 @@
-SELECT name, create_date
```
Comment (pryabov) [inline lab4/task1/file2.sql]: the same

File: stack-to-queue-app/Queue/Impl/QueueStack.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+
+namespace Queue.Impl
+{
+    public class QueueStack<T> : IQueue<T>
+    {
+        private Stack<T> _input = new Stack<T>();
```
Comment (pryabov) [inline stack-to-queue-app/Queue/Impl/QueueStack.cs]: unfortunately `System.Collections` wasn't allowed due initial requirements

File: lab4/task1/file1.sql (hunk 1 above)
Comment (alexander-bedenko) [reply lab4/task1/file1.sql]: It was my previous files from lab4

**Pull request closed without merging**